meta-dts-distro/recipes-dts/dts/dasharo-deploy/dasharo-deploy

```diff
@@ -226,24 +226,26 @@ smbios_migration() {
 }
 
 smmstore_migration() {
-    echo "Beginning SMMSTORE migration process..."
+    echo "Backing up firmware configuration... "
```

Comment:
```suggestion
    echo -n "Backing up firmware configuration... "
```

I wanted to have the Done or Error message printed in the same line to make it a bit shorter and nicer to read

Reply:
@mkopec got it, added in https://github.com/Dasharo/meta-dts/pull/76/commits/dafd803a5b67db2049ff76ea7a746d62caeb4acc